app/src/main/java/com/example/imdb/SignUpActivity.kt

```diff
@@ -25,23 +26,25 @@ class SignUpActivity : AppCompatActivity() {
 
         //Send data to db
         binding.btnAccept.setOnClickListener {
-            if (binding.inputPassword.text.toString().length < 8){
+            if (binding.inputPassword.text.toString().length < 8) {
                 binding.tilPassword.error = "error password"
-            }else{
-                if (emailFound() == null){
+            } else {
+                if (emailFound() == null) {
```

Comment:
emailFound() returns a non-null User object, that's why this is a SENSELESS_COMPARISON

Reply:
I understand !!!